UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -293,5 +308,26 @@ public async Task CreatePosFolderStructure(string largeMediaExchangeSetPath)
             fileSystemHelper.CheckAndCreateFolder(largeMediaExchangeSetAdcPath);
             await Task.CompletedTask;
         }
+
+        private async Task GetReadmeFiles(string batchId, string exchangeSetPath, string correlationId)
```

Comment:
Change the method name please
```suggestion
        private async Task DownloadReadMeFile(string batchId, string exchangeSetPath, string correlationId)
```

Reply:
Changed the method name as suggested